src/SFA.DAS.FAT.Web/Views/Shared/_pageLinks.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using SFA.DAS.FAT.Web.Models.BreadCrumbs
+@using SFA.DAS.FAT.Web.TagHelpers
+@using SFA.DAS.FAT.Web.Infrastructure
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@if (Model is IBackLink backLink)
+{
+    <a asp-show="@Model.ShowBackLink" href="@backLink.BackLinkUrl" title="Find Apprenticeship Training" class="govuk-back-link">Back</a>
```

Comment:
Why are we not using Browser back?

Reply:
you included BackLinkUrl in the original spec, suggesting you wanted to use it